tensorflow/compiler/tf2tensorrt/segment/segment.cc

```diff
@@ -721,6 +721,10 @@ Status SegmentGraph(const Graph* tf_graph,
   std::vector<UnionFind<SimpleNode*>> node_segments;
   for (int i = 0; i < graph->num_node_ids(); ++i) {
     SimpleNode* node = graph->FindNodeId(i);
+	if (!node) {
+        LOG(WARNING) << "Node "<< i <<" not found in Graph";
```

Comment:
Check this code https://github.com/tensorflow/tensorflow/blob/8ed58f3448c6c517f83ead72a9cde9322cb2952c/tensorflow/compiler/tf2tensorrt/segment/segment.cc#L162-L169

This is due to the input graph doesn't have a node for this id.

So I think we shouldn't issue the message as a warning, instead we may issue the message  as:
   VLOG(1) << "Node " << i << " doesn't exist in the graph"

Reply:
Thanks for the comments. Changes done.